tests/llmcompressor/transformers/autoround/test_autoround_oneshot.py

```diff
@@ -38,6 +38,11 @@
         )
     },
 )
+recipe_modifier_nvfp4 = AutoRoundModifier(
```

Comment:
how about adding an example of NVFP4 model quantization, besides adding NVFP4 test?